src/app/app.component.html

```diff
@@ -26,6 +26,11 @@
     </ng-container>
     <ng-container ppw-sidebar-top-content></ng-container>
     <ng-container ppw-page-content>
+        <ppw-breadcrumb>
```

Comment:
So, if people want to add the breadcrumbs to their application, they should add these 5 lines of HTML?

Maybe we want to add this as an option to the ppw-wireframe component? I assume that we will most likely just keep copying these lines from project to project.